src/app/features/home/capture-tab/capture-details/capture-details.page.ts

```diff
@@ -166,6 +166,26 @@ export class CaptureDetailsPage {
       .subscribe();
   }
 
+  openMap() {
+    return this.proof$
+      .pipe(
+        concatMap(proof =>
+          iif(
+            () =>
+              proof.geolocationLatitude !== undefined &&
```

Comment:
For some old data (confirmed to exist on Production account), the `proof.geolocationLatitude` and `proof.geolocationLongitude` is string type `'undefined'` and cannot be excluded with this conditional statement.

It also displays incorrectly in Capture detail page (`undefined, undefined` instead of `Location Not Disclosed`)

This kind of case should be handled